mesonbuild/compilers.py

```diff
@@ -2192,10 +2192,10 @@ def __init__(self, exelist, version, gcc_type, is_cross, exe_wrap, defines):
                           '3': ['-Wall', '-Wpedantic', '-Wextra', '-Winvalid-pch', '-Wnon-virtual-dtor']}
 
     def get_options(self):
-        opts = {'cpp_std' : coredata.UserComboOption('cpp_std', 'C++ language standard to use',
-                                                     ['none', 'c++03', 'c++11', 'c++14', 'c++1z',
-                                                      'gnu++03', 'gnu++11', 'gnu++14', 'gnu++1z'],
-                                                     'none'),
+        opts = {'cpp_std': coredata.UserComboOption('cpp_std', 'C++ language standard to use',
+                                                    ['none', 'c++03', 'c++11', 'c++14', 'c++1z',
+                                                    'gnu++03', 'gnu++11', 'gnu++14', 'gnu++1z'],
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->